app/controllers/katello/api/v2/repositories_controller.rb

```diff
@@ -493,7 +491,7 @@ def upload_content
     param :async, :bool, desc: N_("Do not wait for the ImportUpload action to finish. Default: false")
     param 'publish_repository', :bool, :desc => N_("Whether or not to regenerate the repository on disk. Default: true")
     param 'sync_capsule', :bool, :desc => N_("Whether or not to sync an external capsule after upload. Default: true")
-    param :content_type, RepositoryTypeManager.uploadable_content_types(false).map(&:label), :required => false, :desc => N_("content type ('deb', 'docker_manifest', 'file', 'ostree_ref', 'rpm', 'srpm')")
```

Comment:
We introduced `callable_enum` in Foreman some time ago just for such cases: https://github.com/theforeman/foreman/pull/10494